packages/next/build/babel/plugins/next-page-config.ts

```diff
@@ -52,17 +54,29 @@ export default function nextPageConfig({
                 path: NodePath<BabelTypes.ExportNamedDeclaration>,
                 state: any
               ) {
-                if (
-                  state.bundleDropped ||
-                  !path.node.declaration ||
-                  !(path.node.declaration as any).declarations
-                )
+                if (state.bundleDropped || !path.node.declaration) {
                   return
-                const { declarations } = path.node.declaration as any
+                }
+                const { declarations, id } = path.node.declaration as any
                 const config: PageConfig = {}
 
+                // drop SSR Exports for client bundles
+                if (id && sprSSRExports.has(id.name)) {
+                  if (id.name === 'getStaticProps') {
+                    state.isPrerender = true
+                    sprStatus.used = sprStatus.used || state.isPrerender
```

Comment:
Shouldn't this just be:
```suggestion
                    sprStatus.used = true
```